analysis/monk/src/DampenHarm.tsx

```diff
@@ -0,0 +1,73 @@
+import { formatNumber, formatThousands } from 'common/format';
+import SPELLS from 'common/SPELLS';
+import Analyzer, { Options, SELECTED_PLAYER } from 'parser/core/Analyzer';
+import Events, { DamageEvent } from 'parser/core/Events';
+import BoringSpellValueText from 'parser/ui/BoringSpellValueText';
+import Statistic from 'parser/ui/Statistic';
+import STATISTIC_CATEGORY from 'parser/ui/STATISTIC_CATEGORY';
+import STATISTIC_ORDER from 'parser/ui/STATISTIC_ORDER';
+
+class DampenHarm extends Analyzer {
+  hitsReduced = 0;
+  damageReduced = 0;
+  skip = 0;
```

Comment:
I don't believe this is used at all?
If not remove please

Reply:
Done, was a relic of counting skipped hits with the previous version.